src/views/content/content.scss

```diff
@@ -188,22 +188,6 @@
         }
     }
 
-    .do-header,
```

Comment:
why are we removing this? how does this affect the icons?

Reply:
Previously the check icon is just a check so we need the green background color for the circle. now the circle is part of the icon, so this is not needed.